source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml

```diff
@@ -47,11 +48,24 @@ ComboBox {
         comboBox.popup.open();
     }
 
+    function selectOption(option) {
+	    textField.text = option
+	    comboBox.popup.visible = false
```

Comment:
Should we control this popup via its visibility or its active status?

Reply:
@sachinrajauriya we can do `combobox.popup.close()` here

Reply:
@snbehar I have added the changes.